src/Symfony/Component/Form/Tests/Console/Descriptor/AbstractDescriptorTest.php

```diff
@@ -50,32 +54,64 @@ public function testDescribeResolvedFormType(ResolvedFormTypeInterface $type, ar
         }
     }
 
+    /** @dataProvider getDescribeOptionTestData */
+    public function testDescribeOption(OptionsResolver $optionsResolver, array $options, $fixtureName)
+    {
+        $describedObject = $this->getObjectDescription($optionsResolver, $options);
+        $expectedDescription = $this->getExpectedDescription($fixtureName);
+
+        if ('json' === $this->getFormat()) {
+            $this->assertEquals(json_encode(json_decode($expectedDescription), JSON_PRETTY_PRINT), json_encode(json_decode($describedObject), JSON_PRETTY_PRINT));
+        } else {
+            $this->assertStringMatchesFormat(trim($expectedDescription), trim(str_replace(PHP_EOL, "\n", $describedObject)));
+        }
+    }
+
     public function getDescribeDefaultsTestData()
     {
-        $options['types'] = array('Symfony\Bridge\Doctrine\Form\Type\EntityType');
+        $options['core_types'] = array('Symfony\Component\Form\Extension\Core\Type\FormType');
+        $options['service_types'] = array('Symfony\Bridge\Doctrine\Form\Type\EntityType');
         $options['extensions'] = array('Symfony\Component\Form\Extension\Csrf\Type\FormTypeCsrfExtension');
         $options['guessers'] = array('Symfony\Component\Form\Extension\Validator\ValidatorTypeGuesser');
+        $options['decorated'] = false;
 
         yield array(null, $options, 'defaults_1');
     }
 
     public function getDescribeResolvedFormTypeTestData()
     {
-        $typeExtensions = array(
-            new FormTypeCsrfExtension(new CsrfTokenManager()),
-        );
+        $typeExtensions = array(new FormTypeCsrfExtension(new CsrfTokenManager()));
         $parent = new ResolvedFormType(new FormType(), $typeExtensions);
 
-        yield array(new ResolvedFormType(new ChoiceType(), array(), $parent), array(), 'resolved_form_type_1');
+        yield array(new ResolvedFormType(new ChoiceType(), array(), $parent), array('decorated' => false), 'resolved_form_type_1');
+    }
+
+    public function getDescribeOptionTestData()
+    {
+        $parent = new ResolvedFormType(new FormType());
+        $options['decorated'] = false;
+
+        $resolvedType = new ResolvedFormType(new ChoiceType(), array(), $parent);
+        $options['type'] = $resolvedType->getInnerType();
+        $options['option'] = 'choice_translation_domain';
+        yield array($resolvedType->getOptionsResolver(), $options, 'form_type_option_1');
```

Comment:
Would be great to get rid of number suffixes for test case files and use a short descriptive name instead.

Reply:
Done.